source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Provider Physical Network
+category: feature
+authors: phoracek,amusil
+feature_name: Provider Physical Network
+feature_modules: engine,vdsm,ovn-provider
+feature_status: In Development
+---
+
+# Provider Physical Network
+
+## Summary
+
+This feature extends [ovirt-ovn-provider](https://www.ovirt.org/develop/release-management/features/network/ovirt-ovn-provider/) adding support for physical network access. It allows user to connect external OVN networks to a physical (oVirt native) network.
+
+Feature progress is tracked on [ovn-localnet Trello](https://trello.com/b/lxIBEn0A/ovn-localnet). Related patches can be found on [gerrit topic:localnet](https://gerrit.ovirt.org/#/q/topic:localnet).
+
+### Owner
+
+- Feature Owner: Petr Horáček (phoracek)
+- E-mail: <phoracek@redhat.com>
+
+### Benefit to oVirt
+
+Existing OVS VM networking could be replaced with OVN. That would also allow us to drop a part of VM live migration hook.
```

Comment:
which part is still required? you've already killed the display network part.

Reply:
the VM network part. we need to change OVS bridge name on migration. but since it does not work now and we will drop it (?) we can just drop this note